src/components/mc/cuda/mc_cuda.c

```diff
@@ -301,62 +300,56 @@ static ucc_status_t ucc_mc_cuda_mem_query(const void *ptr,
         return UCC_OK;
     }
 
-    if (ptr == 0) {
```

Comment:
Why check for NULL is removed?

Reply:
moved to ucc_mc_get_mem_attr instead, previously it was same code in cuda and cpu mem components